### PR TITLE
feat(website): add table of content sidebar

### PR DESCRIPTION
**Pull request by emmenko**

### CODE REVIEW

File: website/src/layouts/content.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { css } from '@emotion/core';
 import LayoutContainer from './internals/layout-container';
 import LayoutHeader from './internals/layout-header';
 import LayoutSidebar from './internals/layout-sidebar';
+import LayoutToc from './internals/layout-toc';
```
Comment (montezume) [inline website/src/layouts/content.js]: nitpick: LayoutTOC

File: website/src/layouts/internals/layout-toc.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  color: ${colors.light.text};
+  text-decoration: none;
+
+  :hover {
+    color: ${colors.light.primarySoft};
+    text-decoration: underline;
+  }
+  :hover,
+  :active {
+    outline-width: 0;
```
Comment (montezume) [inline website/src/layouts/internals/layout-toc.js]: 😢

Comment (emmenko) [reply website/src/layouts/internals/layout-toc.js]: ?

Comment (montezume) [reply website/src/layouts/internals/layout-toc.js]: Why are we removing the outline? makes it really hard to use keyboard to navigate

Comment (emmenko) [reply website/src/layouts/internals/layout-toc.js]: The focus outline for a11y is still there, this is just for hover/active state.

**Pull request merged**